community/resources/resources.md

```diff
@@ -37,14 +37,19 @@ We have channels for each project. These channels are recommended for new member
 - [github][]: Support for GitHub issues, pull requests, and processes
 - [public-backlog][]: Discussions of the Magento 2 backlog
 - [devdocs][]: Documentation contribution support
+- [cloud][]: Open chat for cloud questions
```

Comment:
I don't know if the purpose of these chat is correctly described, maybe @jeff-matthews (or magento engcom pip ) can help with the people in charge of the channels

Reply:
Ideally, we could copy/paste the text from the "channel purpose" in Slack. Unfortunately, very few channels have that property set in a meaningful way.

Deferring to @lorikrell for this one since I know she's very active in those channels supporting the EngComm team.

Reply:
@osrecio sorry for late reply! I just posted this page for Community Engineers to easily update/add new channels. https://github.com/magento/magento2/wiki/Slack-Channels Could be an excellent replacement!